### PR TITLE
Account for custom servlet context paths in GraphiQL callback

### PR DESCRIPTION
**Pull request by bono007**
Fixes gh-171

@paulbakker @berngp @jandrade1

### CODE REVIEW

**Review by jandrade1 — APPROVED** [review]
Looks good to me @bono007 , thanks for working on this so quickly!

File: graphql-dgs-spring-webmvc-autoconfigure/src/test/kotlin/com/netflix/graphql/dgs/webmvc/autoconfigure/GraphiQLPathConfigWithCustomGraphQLPathTest.kt [hunk 1]
```diff
@@ -27,7 +27,7 @@ import org.springframework.boot.test.web.client.TestRestTemplate
     webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
     properties = ["dgs.graphql.path=/zuzu"]
 )
-class GraphiQLPathConfigCustomGraphQLPathTest(@Autowired val restTemplate: TestRestTemplate) {
+class GraphiQLPathConfigWithCustomGraphQLPathTest(@Autowired val restTemplate: TestRestTemplate) {
```
Comment (bono007) [inline graphql-dgs-spring-webmvc-autoconfigure/src/test/kotlin/com/netflix/graphql/dgs/webmvc/autoconfigure/GraphiQLPathConfigWithCustomGraphQLPathTest.kt:30]: **[UNRELATED]** Renamed the GraphiQL config tests to clearly indicate the scenario being tested.

File: graphql-dgs-spring-webmvc-autoconfigure/src/main/kotlin/com/netflix/graphql/dgs/webmvc/autoconfigure/GraphiQLConfigurer.kt [hunk 2]
```diff
@@ -49,13 +51,14 @@ open class GraphiQLConfigurer(
     }
 
     override fun addResourceHandlers(registry: ResourceHandlerRegistry) {
+        val graphqlPath = servletContext.contextPath + configProps.path
```
Comment (berngp) [inline graphql-dgs-spring-webmvc-autoconfigure/src/main/kotlin/com/netflix/graphql/dgs/webmvc/autoconfigure/GraphiQLConfigurer.kt:54]: Info logging that hints the developer where the endpoint is available?

**Review by berngp — APPROVED** [review]
Thanks! 
Nit: some logging to hint where the endpoint is available.

**Comment by bono007** [comment]
Great idea.  I'll add in a bit.

File: graphql-dgs-spring-webmvc-autoconfigure/src/main/kotlin/com/netflix/graphql/dgs/webmvc/autoconfigure/GraphiQLConfigurer.kt (hunk 2 above)
Comment (bono007) [reply graphql-dgs-spring-webmvc-autoconfigure/src/main/kotlin/com/netflix/graphql/dgs/webmvc/autoconfigure/GraphiQLConfigurer.kt:58]: Done.

**Review by jandrade1 — APPROVED** [review]

**Pull request merged**